aries_cloudagent/core/util.py

```diff
@@ -149,7 +169,9 @@ async def get_version_def_from_msg_class(
         if version_definition:
             return version_definition
     definition_path = _get_path_from_msg_class(msg_class)
+    print(f"definition_path = {definition_path}")
     version_definition = _get_version_def_from_path(definition_path, major_version)
+    print(f"version_definition = {version_definition}")
```

Comment:
Couple of prints that we probably don't want.

Reply:
good catch, thanks. removed.